contracts/Token.sol

```diff
@@ -26,7 +31,21 @@ contract Token is Initializable, IToken, IERC20 {
   mapping (address => AccountLib.Data) accounts;
   mapping (address => mapping (address => uint256)) allowances;
 
-  // Public functions.
+  /// --- Constants.
```

Comment:
@afmfe-iul The way `ERC1404` works is that before every transfer, you call a function to "detect" (`detectTransferRestriction`) whether you'd be authorised to perform it. These tests should not cover balance etc, only business logic that imposes restrictions (Being an actor etc).

The detection function returns errors in the form of `uint8` numbers, and exposes another function to allow a code to be mapped into a human-readable string message (`messageForTransferRestriction`).

These are essentially constants for each error in our transfer restriction logic, and their message counterpart.